build.sbt

```diff
@@ -16,6 +16,12 @@ val `sbt-scalajs-bundler` =
       description := "Module bundler for Scala.js projects",
       libraryDependencies += "com.typesafe.play" %% "play-json" % "2.6.7",
       addSbtPlugin("org.scala-js" % "sbt-scalajs" % scalaJSVersion),
+      libraryDependencies ++= {
+        if (isScalaJS1x)
+          List("org.scala-js" %% "scalajs-linker" % scalaJSVersion)
```

Comment:
So, a possible drawback could be that this could conflict with other build dependencies? Do you think we could find a way to not have to implement this proxy over the Scala.js linker?

Reply:
It is a possible drawback, but that drawback already exists today with sbt-scalajs 0.6.x and with scalajs-bundler for 0.6.x, because before sbt-scalajs depended itself on scalajs-linker.

We should find a way to improve this but it really should be a different PR.